.github/workflows/coverage.yaml

```diff
@@ -62,3 +69,18 @@ jobs:
               body: `### Coverage report
               **Main**: ${main.toFixed(2)}% | **PR**: ${pr.toFixed(2)}% | **Diff: ${diff.toFixed(2)} ${diff >= 0 ? '✅' : '⚠️'}**`
             })
+      - env:
+          pr: ${{ fromJson(needs.coverage.outputs.cov-pr).totals.percent_covered }}
+          main: ${{fromJson(needs.coverage.outputs.cov-main).totals.percent_covered}}
+        run: |
+          diff=$(echo "$pr - $main" | bc)
+          main_fixed=$(echo "scale=2; $main/1" | bc)
+          pr_fixed=$(echo "scale=2; $pr/1" | bc)
+          diff_fixed=$(echo "scale=2; $diff/1" | bc)
+          if [ $(echo "scale=0; $diff/1" | bc) -gt 0 ]; then
+            icon='✅'
+          else
+            icon='⚠️'
+          fi
+          echo "### Coverage report
+            **Main**: ${main_fixed}% | **PR**: ${pr_fixed}% | **Diff: ${diff_fixed} ${icon}**" >> "$GITHUB_STEP_SUMMARY"
```

Comment:
It's theoretically possible to write this in github-script, but I tried a couple of times and I gave up. I don't speak github-script (I do sometimes speak javascript/typescript, and I contribute to their various repositories, but I generally end up using shell/perl to talk to github apis instead).

For the curious, you can review [my attempts at making this workflow behave](https://github.com/jsoref/temporian/actions/workflows/coverage.yaml?query=branch%3Afix-coverage-workflow).

At posting, this is the [working one].(https://github.com/jsoref/temporian/actions/runs/8097376074#summary-22128559880)

Think of this more as a proposal than a "definitely merge this approach". If you have someone who's familiar w/ github-script, then they should be able to make a simple fix to https://github.com/jsoref/temporian/commit/64a0753ec2a65f963f1c528f6813b7bb3d09d322 (which didn't work -- see the lack of a summary in https://github.com/jsoref/temporian/actions/runs/8097265388).